sites/partners/__tests__/pages/users/index.test.tsx

```diff
@@ -110,16 +109,16 @@ describe("users", () => {
     const header = await findByText("Partners Portal")
     expect(header).toBeInTheDocument()
     expect(getByText("Add User")).toBeInTheDocument()
-    expect(getByText("Export")).toBeInTheDocument()
-    fireEvent.click(getByText("Export"))
-    jest.clearAllTimers()
+    const exportButton = await findByText("Export")
+    expect(exportButton).toBeInTheDocument()
+    fireEvent.click(exportButton)
     const successMessage = await findByText("The File has been exported")
     expect(successMessage).toBeInTheDocument()
   })
 
   it("should render error message csv fails", async () => {
     // set a logged in token
-    jest.useFakeTimers()
+    jest.spyOn(console, "log").mockImplementation(jest.fn())
```

Comment:
Spying on a log seems a little close to implementation details, if the error message is in the document it seems like we might not need the spy? I might be missing its purpose

Reply:
I don't love doing this, but this is a way for the test output to be a little less cluttered. This test purposely throws an error so an error  is expected, however when it appears in the test logs it looks like an error happened in the test even though the test passes. So this is just to make it so that those logs are hidden. However, I'm ok not doing this since it does all mute all console logging in the test